### PR TITLE
Add NSHE theme

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
Adds new NSHE theme ("ons-nhs" in Runner) and updates existing logos to align with latest DS changes.

### How to review
New unit tests (cases) added where appropriate, validator PR also created to support new theme validation. New schema added, new functional test.

### Checklist
* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

File: schemas/test/en/test_theme_ons_nhs.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Test ONSNHS ONS Survey",
+    "theme": "ons-nhs",
+    "description": "A questionnaire to demo the UKHSA ONS survey theme",
```
Comment (rmccar) [inline schemas/test/en/test_theme_ons_nhs.json:9]:
```suggestion
    "description": "A questionnaire to demo the ons-nhs survey theme",
```

File: schemas/test/en/test_theme_ons_nhs.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                                    }
+                                ],
+                                "id": "radio-question",
+                                "title": "What is your favourite breakfast food?",
+                                "type": "General"
+                            }
+                        }
+                    ],
+                    "id": "group",
+                    "title": "UKHSA ONS Theme Test"
```
Comment (rmccar) [inline schemas/test/en/test_theme_ons_nhs.json:70]:
```suggestion
                    "title": "ONS NHS Theme Test"
```

File: tests/functional/spec/theme_nhse.spec.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import RadioPage from "../generated_pages/theme_ons_nhs/radio.page";
+import { expect } from "@wdio/globals";
+
+describe("Theme NHSE", () => {
```
Comment (rmccar) [inline tests/functional/spec/theme_nhse.spec.js:4]: Is NHSE the name of the theme? Shouldn't it be ONS-NHS?

File: templates/assets/images/desnz-logo-stacked.svg [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<svg class="ons-u-di ons-u-mr-m" xmlns="http://www.w3.org/2000/svg" focusable="false" width="105" height="56" viewBox="0 0 105 56" aria-labelledby="desnz-logo-alt">
```
[the remark this answers is not in the document]
Comment (petechd) [reply templates/assets/images/desnz-logo-stacked.svg:1]: Can we exclude the change from this PR? Will it still work as expected?

[the remark this answers is not in the document]
Comment (rmccar) [reply templates/assets/images/desnz-logo-stacked.svg:1]: Yeah that would be fine, it'll still work as expected just think we should space them all using the classes can leave it like this for now

File: tests/functional/spec/theme_nhse.spec.js (hunk 3 above)
Comment (rmccar) [reply tests/functional/spec/theme_nhse.spec.js:4]: Feels like a typo to me but yeah can see its down as NHSE in multiple cases on the card

**Review by rmccar — APPROVED** [review]

**Review by berroar — APPROVED** [review]

File: scripts/run_validator.sh [hunk 5]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
-tag=latest
+tag=add-nhse-theme
```
Comment (VirajP1002) [inline scripts/run_validator.sh:2]: Will this need to be reverted to latest?

Comment (petechd) [reply scripts/run_validator.sh]: Reverted now

**Pull request merged**